Conan.VisualStudio.Core/ConanProject.cs

```diff
@@ -1,11 +1,31 @@
 using System.Collections.Generic;
+using System.IO;
+using Newtonsoft.Json.Linq;
+using System;
 
 namespace Conan.VisualStudio.Core
 {
     public class ConanProject
     {
         public string Path { get; set; }
-
+        public string ConfigFile { get; set; }
         public List<ConanConfiguration> Configurations { get; } = new List<ConanConfiguration>();
+
+        public string getProfile(ConanConfiguration configuration)
+        {
+            if (!File.Exists(ConfigFile)) return null;
+            try
+            {
+                JObject jObject = JObject.Parse(File.ReadAllText(ConfigFile));
+                var configs = jObject["configurations"].ToObject<Dictionary<string, string>>();
+                configs.TryGetValue(configuration.VSName, out string conanProfile);
+                return conanProfile;
```

Comment:
We should write a warning message (_"File `conan.config.json` file available for the project, but no profile declared for VS configuration 'xxxx'"_), but I'm not sure how are we handling the output. @SSE4, can you point me in the right direction?

Reply:
we have two ways to output messages.
1. `ErrorList` Window - use `ErrorListService` for this
2. `Output` Window - use `Logger` class
I think for this one it's better to use ErrorList, and double click on error may open `conan.config.json` with incorrect line

Reply:
But I don't have the `ErrorListService` in the `Core` project...

Reply:
then it needs to get delegate/callback function for error reporting

Reply:
probably the _Interface_ in the `Core` and the implementation in the other project, I've read more about this pattern than using _callbacks_, right?

Reply:
something like `IAbstractErrorReporting` interface which might be passed to the path helper. then `ErrorListService`, `DialogService` or `OutputterService` (class to write into output window) may inherit from this interface, and could be passed into the core.

Reply:
Done 👍